cmd/notation/sign.go

```diff
@@ -6,16 +6,16 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/notaryproject/notation-core-go/timestamp"
 	"github.com/notaryproject/notation-go"
-	"github.com/notaryproject/notation-go/crypto/timestamp"
 	"github.com/notaryproject/notation/internal/cmd"
 	"github.com/notaryproject/notation/internal/envelope"
 	"github.com/spf13/cobra"
 )
 
 type signOpts struct {
 	cmd.SignerFlagOpts
-	RemoteFlagOpts
+	SecureFlagOpts
 	timestamp       string
 	expiry          time.Duration
 	originReference string
```

Comment:
Do we need `originReference` and `timestamp`?

Reply:
No, we don't. Will update the code and remove them.